_ja/overviews/scala3-migration/compatibility-intro.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 互換性リファレンス
+type: chapter
+description: この章ではScala 2.13とScala 3の互換性について説明しています。
+num: 1
+previous-page:
+next-page: compatibility-source
+language: ja
+---
+
+Scala 3はScalaエコシステムにおける互換性に関して革新的なものであり、全Scalaプログラマの日々の経験を大きく向上させるでしょう。
+この新しい互換性の時代は移行から始まります。
+
+Scala 2からScala 3に映ることは大きな急速進歩であります。
+Scalaはピカピカの新しいコンパイラで、言語のコアな基礎部分の完全なる再デザインを踏まえています。まだ、Scala 2.12からScala 2.13への移行のときよりは、ハードじゃないことを主張したいです。
+
+Scala 2.13とScala 3の間での相互互換性のおかげで一層シンプルになりました。
```

Comment:
```suggestion
Scala 2.13 と Scala 3 の間での相互互換性のおかげで、見方によっては以前よりもシンプルになったと言えるかもしれない。
```

原文: It will even be simpler in some respects, thanks to the interoperability between Scala 2.13 and Scala 3.

"in some respects" は「見方によっては」的なちょっと防衛線を張った言い方なのでそれを反映させました。